### ISSUE TITLE
Automatic documentation builds didn't notice version changes

### ISSUE THREAD

**Issue by lcawl** [elastic/docs#1792]
Related to https://github.com/elastic/docs/issues/804

On the day of the 6.8.8 and 7.6.2 releases, I merged https://github.com/elastic/docs/pull/1778 which updates the version numbers for the documentation.

By my understanding, the purpose of listing the {version}.asciidoc in the sources section of our books (e.g. https://github.com/elastic/docs/blob/9b9488223801dce7c340b1e47c7a5f8c2ed11062/conf.yaml#L315) was to cause those books to be rebuilt when the version changed.

However, the next two automatic documentation builds failed to notice the version changes. I ultimately had to force a rebuild of all versions (which takes longer on time-critical release days) to force the new versions to be picked up and shown in pages like these: 
- https://www.elastic.co/guide/en/kibana/6.8/targz.html (still showed 6.8.7 instead of 6.8.8)
- https://www.elastic.co/guide/en/elasticsearch/reference/7.6/targz.html (still showed 7.6.1 instead of 7.6.2)

At the time of the problem, @nik9000 verified that http://master.docs-preview.app.elstc.co/guide/en/kibana/6.8/targz.html and https://github.com/elastic/built-docs/blob/master/raw/en/kibana/6.8/targz.html were also not showing the updated versions.

Is the detection of version changes not working in our automatic builds after all?

**Comment by gtback** [comment]
Yep! As I understand it, that's why we include those `sources` in `conf.yaml`. I'll have to look into this more.
